packages/flutter/lib/src/widgets/text_selection.dart

```diff
@@ -3651,29 +3349,32 @@ mixin TextSelectionHandleControls on TextSelectionControls {
     TextSelectionDelegate delegate,
     ValueListenable<ClipboardStatus>? clipboardStatus,
     Offset? lastSecondaryTapDownPosition,
-  ) => const SizedBox.shrink();
+  ) =>
+      const SizedBox.shrink();
 
-  @override
+  /// Returns true if the cut action is enabled on this toolbar.
```

Comment:
does this actually do anything at this point?

Reply:
No, I've now added deprecations here too as suggested by @Piinks in https://github.com/flutter/flutter/pull/141310#discussion_r1464021613.